sub-packages/bionemo-evo2/src/bionemo/evo2/run/train.py

```diff
@@ -575,30 +835,20 @@ def train(args: argparse.Namespace) -> nl.Trainer:
             }
         else:
             activation_checkpointing_args = {}
+
     # Retrieve model config.
     config_modifiers_init = {
-        "calculate_per_token_loss": not args.no_calculate_per_token_loss,  # override megatron internal behavior.
         "tp_comm_overlap": args.use_megatron_comm_overlap_llama3_8k,
         "seq_length": args.seq_length,
         "hidden_dropout": args.hidden_dropout,
         "attention_dropout": args.attention_dropout,
-        "to_upper": "weighted" if args.no_renormalize_loss else "normalized_weighted",
```

Comment:
This can move into the mamba/hyena configs, or you could pop it out of the llama config?